docker-compose.yml

```diff
@@ -11,11 +11,13 @@ services:
       SEC_CRYPTO_SERVICE_BASE_URI: ${SEC_CRYPTO_SERVICE_BASE_URI}
       SEC_CRYPTO_SERVICE_ENDPOINT_SIGN_PATH: ${SEC_CRYPTO_SERVICE_ENDPOINT_SIGN_PATH}
       SEC_USE_CERTFICATES: ${SEC_USE_CERTFICATES}
-      SEC_KEY_STORE_PATH: ${SEC_KEY_STORE_PATH}
+      #SEC_KEY_STORE_PATH: ${SEC_KEY_STORE_PATH}

```

Comment:
Is this commented out because it is optional?

Reply:
It is, there's a default path in there that I took advantage of in my testing. This docker-compose isn't really being ran anywhere anyway, just used for testing things locally.

Reply:
okay, just making sure this wasn't accidentally left in if it was meant to be taken out